include/seastar/core/future.hh

```diff
@@ -216,12 +221,10 @@ struct broken_promise : std::logic_error {
 /// This is equivalent to
 /// make_exception_future(std::current_exception()), but expands to
 /// less code.
+SEASTAR_MODULE_EXPORT
 template <typename T = void>
 future<T> current_exception_as_future() noexcept;
 
-extern template
-future<void> current_exception_as_future() noexcept;
-
```

Comment:
Why delete this? It will waste some compile time.

Reply:
i forgot to export it as well. as all visibility properties of the same symbol should agree, if any of them are different with the others, compiler just chokes. added, and put the declaration back. the tree compiles.